project/src/js/ui/tournament/detail/tournamentDetailCtrl.js

```diff
@@ -1,56 +1,83 @@
 import socket from '../../../socket';
+import socketHandler from './socketHandler';
 import * as utils from '../../../utils';
 import * as xhr from '../tournamentXhr';
 import helper from '../../helper';
 import m from 'mithril';
 
 export default function controller() {
-  const id = m.route.param('id');
+	helper.analyticsTrackView('Tournament details');
 
-  helper.analyticsTrackView('Tournament details');
+	const tournament = m.prop();
+	const hasJoined = m.prop(false);
 
-  const tournament = m.prop();
+	function reload(data) {
```

Comment:
Not sure what happened with the indenting. It looked fine to me in Atom and eslint didn't complain. I'll figure out what's happening and fix.